config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java

```diff
@@ -298,6 +298,20 @@ public void testNoOverride() {
     assertThat(config.getConfigOptions().getEvmStackSize()).isNotPresent();
   }
 
+  @Test(expected = RuntimeException.class)
```

Comment:
We are transitioning away from `expected` in annotation to `assertThatExceptionOfType(...).isThrownBy(...).withMessage(...)`.  It reads better and makes it more clear where the exception will occur. Can you migrate to this format?

Reply:
as seen here - https://github.com/hyperledger/besu/blob/98ec5777dd61aa006727611644329b0195a939cd/enclave/src/test/java/org/hyperledger/besu/enclave/EnclaveFactoryTest.java#L44